easybuild/easyconfigs/g/GeneTorrent/GeneTorrent-3.8.7-foss-2015a.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+
+name = 'GeneTorrent'
+version = '3.8.7'
+
+homepage = 'https://cghub.ucsc.edu/index.html'
+description = """ GeneTorrent is a command-line tool required to download datasets
+held in repositories at the Cancer Genomics Hub. """
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['https://cghub.ucsc.edu/software/downloads/GeneTorrent/%(version)s/']
+sources = ['GeneTorrent-%(version)s.tar.gz']
+
+configopts = '--with-boost=$EBROOTBOOST --with-xerces=$EBROOTXERCESMINCPLUSPLUS --with-xqilla=$EBROOTXQILLA'
+
+dependencies = [
+    ('Xerces-C++', '3.1.2'),
+    ('cURl', '7.44.0'),
+    ('XQilla', '2.3.0'),
+    ('Boost', '1.55.0', '-Python-2.7.9'),
```

Comment:
The requirement is for >=1.48, so I was vaguely trying to use one nearer to that. I don't have any objection to a newer version, if it works. 

I won't be able to bring in proper user testing until tomorrow i.e. actually downloading files. I've tested that the binary is installed and --version works.